evm_loader/lib/src/commands/emulate.rs

```diff
@@ -66,19 +69,58 @@ pub async fn execute<T: Tracer>(
         .as_ref()
         .and_then(|t| t.state_overrides.clone());
 
+    let solana_overrides = emulate_request.solana_overrides.map(|overrides| {
+        overrides
+            .iter()
+            .map(|(pubkey, account)| (*pubkey, account.as_ref().map(Account::from)))
+            .collect()
+    });
+
     let mut storage = EmulatorAccountStorage::with_accounts(
         rpc,
         program_id,
         &emulate_request.accounts,
         emulate_request.chains,
         block_overrides,
         state_overrides,
+        solana_overrides,
     )
     .await?;
 
-    let step_limit = emulate_request.step_limit.unwrap_or(100_000);
+    let step_limit = emulate_request.step_limit.unwrap_or(100000);
+
+    let result = emulate_trx(emulate_request.tx.clone(), &mut storage, step_limit, tracer).await?;
+
+    if storage.is_timestamp_used() {
+        let mut storage2 = EmulatorAccountStorage::new_from_other(&storage, 5, 3);
+        if let Ok(result2) = emulate_trx(
+            emulate_request.tx,
+            &mut storage2,
+            step_limit,
+            Option::<T>::None,
+        )
+        .await
+        {
+            let response = &result.0;
+            let response2 = &result2.0;
```

Comment:
Why do you take half the values from the first emulation?

Reply:
I return the result from the first emulation and used resources as a combination from the first and second emulations.
Fixed `solana_accounts` list